rust/src/sip/sip.rs

```diff
@@ -277,6 +278,7 @@ impl SIPState {
                         tx.response_line = resp_line;
                     }
                     self.transactions.push_back(tx);
+                    sc_app_layer_parser_trigger_raw_stream_inspection(flow, Direction::ToServer as i32);
```

Comment:
Hi Shivani,
Shouldn't `Direction::ToClient` be set here, since a sip response is being parsed?